front/app/containers/Admin/messaging/texting/ViewCreatedMessage/index.tsx

```diff
@@ -0,0 +1,217 @@
+import React, { useState, useEffect } from 'react';
+
+// components
+import TextArea from 'components/UI/TextArea';
+import Button from 'components/UI/Button';
+import { Label, Box, IconTooltip } from '@citizenlab/cl2-component-library';
+import { Section, SectionField } from 'components/admin/Section';
+import HelmetIntl from 'components/HelmetIntl';
+import TextingHeader from '../components/TextingHeader';
+import FormattedStatusLabel from '../components/FormattedStatusLabel';
+
+// i18n
+import { FormattedDate } from 'react-intl';
+
+// utils
+import clHistory from 'utils/cl-router/history';
+import { withRouter, WithRouterProps } from 'react-router';
+
+// hooks
+import useTextingCampaign from 'hooks/useTextingCampaign';
+
+// services
+import {
+  updateTextingCampaign,
+  ITextingCampaignData,
+  ITextingCampaignStatuses,
+} from 'services/textingCampaigns';
+
+// styling
+import styled from 'styled-components';
+import { isNilOrError } from 'utils/helperUtils';
+
+const StyledForm = styled.form`
+  width: 500px;
+`;
+
+// enough to fit 3 messages, actual functionality TBD in subsequent ticket
+const MAX_CHAR_COUNT = 480;
+
+const getTitleMessage = (campaignStatus: ITextingCampaignStatuses) => {
+  switch (campaignStatus) {
+    case 'draft':
+      return 'Draft SMS Campaign';
+    case 'sending':
+      return 'Sending SMS Campaign';
+    case 'sent':
+      return 'Sent SMS Campaign';
+    case 'failed':
+      return 'Failed SMS Campaign';
+    default:
+      return 'Created SMS Campaign';
+  }
+};
+
+const getAdditionalInfoByStatus = (campaign: ITextingCampaignData) => {
+  switch (campaign.attributes.status) {
+    case 'draft':
+      return (
+        <>
+          Created at: <FormattedDate value={campaign.attributes.created_at} />
+        </>
+      );
+    case 'sending':
+      return (
+        <>
+          Sent at: <FormattedDate value={campaign.attributes.sent_at} />
+        </>
+      );
+    case 'sent':
+      return (
+        <>
+          Sent at: <FormattedDate value={campaign.attributes.sent_at} />
+        </>
+      );
+    case 'failed':
+      return (
+        <>
+          Created at: <FormattedDate value={campaign.attributes.created_at} />
+        </>
+      );
+    default:
+      return (
+        <>
+          Created at: <FormattedDate value={campaign.attributes.created_at} />
+        </>
+      );
+  }
+};
+
+const ViewCreatedMessage = (props: WithRouterProps) => {
+  const [inputPhoneNumbers, setInputPhoneNumbers] = useState('');
```

Comment:
Isn't this an array of strings?

Reply:
This is something we'll need to follow-up on in subsequent PRs I think. The textarea right now returns just a big string, which we process into an array of strings for phone numbers for the BE. We'll need to work with BE to make sure we're treating the data the same in both directions, like when we get a draft message we display the array of strings as a comma-separated string in the text area.